.pipelines/scripts/verify_shell.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+set -euo pipefail
+
+installed=$(which shellcheck 2>&1 >/dev/null)
+if [[ ${installed} -ne 0 ]]; then
+    echo "shellcheck not installed...trying to install."
+    DISTRO=$(uname | tr "[:upper:]" "[:lower:]")
+    if [[ "${DISTRO}" == "ubuntu" ]]; then
+        apt-get install shellcheck
+    elif [[ "${DISTRO}" == "darwin" ]]; then
+        brew install cabal-install
+    else 
+        echo "distro ${DISTRO} not supported at this time. skipping shellcheck"
+        return
+    fi
+fi
+
+filesToCheck=$(find . -type f -name "*.sh" -not -path './parts/linux/cloud-init/artifacts/*')
+
+echo "Running shellcheck..."
+
+IGNORED="
```

Comment:
need to figure out which errors/warning to ignore

Reply:
have you tried those in aks-engine? https://github.com/Azure/aks-engine/blob/f2a577d0ab550b769d5abaf276e45e8e6f95bfdd/scripts/validate-shell.sh#L17

Reply:
ah yea of course...i'll get those in :)